src/style.css

```diff
@@ -0,0 +1,113 @@
+* {
+  margin: 0;
+}
+
+.card {
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.iMac {
+  padding-top: 32px;
+  padding-left: 19px;
+  width: 160px;
+}
+
+.iMacMark {
+  width: 166px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  padding-top: 40px;
+  padding-left: 16px;
+}
+
+.iMacCode {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.rate {
+  display: flex;
+  width: 166px;
+  padding-top: 16px;
+  padding-left: 16px;
+}
+
+.stars {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars:nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
logic with stars should work correctly (like on hw with stars): if you add class stars--3, there should be 3 active stars